airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -23,13 +23,19 @@
 #
 
 from enum import Enum
-from string import Template
 from typing import Any, List, Mapping
 
 from google.ads.googleads.client import GoogleAdsClient
-from google.ads.googleads.v7.services.types.google_ads_service import GoogleAdsRow, SearchGoogleAdsResponse
+from google.ads.googleads.v8.services.types.google_ads_service import GoogleAdsRow, SearchGoogleAdsResponse
+from proto.marshal.collections import Repeated, RepeatedComposite
 
-REPORT_MAPPING = {"ad_group_ad_report": "ad_group_ad"}
+REPORT_MAPPING = {
+    "ad_group_ad_report": "ad_group_ad",
+    "accounts": "customer",
+    "campaigns": "campaign",
+    "ad_groups": "ad_group",
+    "ad_group_ads": "ad_group_ad",
```

Comment:
is this the same as the `ad_group_ad_report`? if so should we just rename the former?

Reply:
I don't think they are the same thing. The differences between these streams are detailed below.

Reply:
Fundamentally yes. But `ad_group_ad_report` in this case is presented as [Ad Performance Report](https://developers.google.com/google-ads/api/docs/migration/mapping#ad_performance). This stream gives us not only information about `Google Ads`, but also statistical and metric data that we can receive daily.
If we combine these two streams, then the essence of Google Ads will cease to exist as such, and we will receive one record for each Google Ad every day, which contains all the information about Ad and a lot of daily statistics. In this case, the user may simply not understand how many Ads he has, and he will also not be able to work only with Ads, he will have to work only with their reports.